apps/pie-docs/src/assets/styles/components/_nav.scss

```diff
@@ -5,105 +5,108 @@ $nav-category-icon-width: f.spacing(e); // this needs to be kept in sync with th
 $nav-item-left-margin: calc($nav-category-gap + $nav-category-icon-width);
 
 .c-nav {
-  @include f.font-size('body-s');
-  background: f.$color-container-default;
-  padding-top: f.spacing(d);
-
-  ul {
-    list-style-type: none;
-    margin: 0;
-    padding: 0;
-  }
-
-  hr {
-    margin: 0;
-  }
-
-  @include f.media('>=wide') {
-    border-right: 1px solid f.$color-border-default;
-  }
-
-  @include f.media('<wide') {
-    max-width: 80%;
-    position: relative;
-    z-index: f.zIndex(high);
-  }
-
-  .c-nav-list {
-    margin: f.spacing(a) 0 f.spacing(b) 0;
-  }
-}
-
-.c-nav-item:hover,
-.c-nav-item:focus,
-.c-nav-category:hover,
-.c-nav-category:focus {
-  background-color: f.$color-container-subtle;
-  text-decoration: underline;
-}
-
-// Hide the default summary icon
-.c-nav-category::-webkit-details-marker,
-.c-nav-category::marker {
-  display: none;
-  content: '';
+    @include f.font-size('body-s');
+    background: f.$color-container-default;
+    padding-top: f.spacing(d);
+
+    @include f.media('>=wide') {
+        border-right: 1px solid f.$color-border-default;
+    }
+
+    @include f.media('<wide') {
+        position: absolute;
+        top: var(--header-height);
+        height: calc(100% - var(--header-height));
+        height: calc(100dvh - var(--header-height));
```

Comment:
Do we need both of these rules?

Reply:
Yeh, I can add a comment if it might help.

Basically, the first rule is a fallback for when `dvh` isn't supported, as it's quite a new CSS feature.

Reply:
Added